feat: new blog options

abaicus opened the pull request:
### Summary
Adds new blog options mentioned inside Codeinwp/neve-pro-addon#852.
<!-- Please describe the changes you made. -->

### Will affect visual aspect of the product
<!-- It includes visual changes? -->
NO - it shouldn't.

### Test instructions
<!-- Describe how this pull request can be tested. -->

- Should migrate fine from the old version settings
- All new settings should work as expected

<!-- Issues that this pull request closes. -->
Ref Codeinwp/neve-pro-addon#852.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

irinelenache commented:
@abaicus Tested it and found this so far:

- [ ] When setting the layout to Grid (one column) and adding Thumbnail shadow, if the picture has lower resolution the shadow appears like the picture would be wider.
https://prnt.sc/uhnor4
https://prnt.sc/uhnt6d

- [ ] Don't know if this is relevant but when i disable the meta elements, they are placed in the same order, no matter in which order i disable them or which order they had before.
https://www.loom.com/share/6bb8ab21843845cd9f0353325649394c

abaicus commented:
@irinelenache 

Thanks for taking the time to look at this.

>  When setting the layout to Grid (one column) and adding Thumbnail shadow, if the picture has lower resolution the shadow appears like the picture would be wider.

Adjusted this in the latest commits :+1: 

---

> Don't know if this is relevant but when i disable the meta elements, they are placed in the same order, no matter in which order i disable them or which order they had before.

That's how it works :+1:

rodica-andronache commented:
@abaicus in RTL mode, in Customizer, the range controls looks a bit cut off
![1](https://user-images.githubusercontent.com/3266155/93489085-c4cfcd80-f90f-11ea-9e1e-f2cce0f91e0b.png)

compared to the current version
![2](https://user-images.githubusercontent.com/3266155/93489087-c5686400-f90f-11ea-8f6c-71e42aa9652d.png)

rodica-andronache commented:
@abaicus saw the same issue from here https://github.com/Codeinwp/neve/issues/1982. Masonry doesn't work in Safari.

Safari
<img width="1150" alt="safari" src="https://user-images.githubusercontent.com/3266155/93807622-8ad63280-fc53-11ea-8ff6-8ea5f049414a.png">

Chrome
![chrome](https://user-images.githubusercontent.com/3266155/93807640-91fd4080-fc53-11ea-8072-6f9647384b8c.png)

rodica-andronache commented:
In `Cover` mode, is the `Thumbnail` from `Post Content Order` supposed to do anything? I was thinking that maybe when that's disabled, the image won't be displayed as the background cover anymore, but right now changing its position/enabling/disabling doesn't do anything

rodica-andronache commented:
Imported the theme unit test.
In `Cover` mode, some posts don't look as good as they do in `Grid` for example:

Grid
![1](https://user-images.githubusercontent.com/3266155/93808915-49df1d80-fc55-11ea-934a-354f4458e917.png)

Cover
![2](https://user-images.githubusercontent.com/3266155/93808906-48155a00-fc55-11ea-9199-845c3f2464d7.png)

Grid
![download](https://user-images.githubusercontent.com/3266155/93809024-6d09cd00-fc55-11ea-9b7b-05f8d1f85056.png)

Cover
![download (1)](https://user-images.githubusercontent.com/3266155/93809036-72ffae00-fc55-11ea-9059-94fa0bbcc20e.png)

abaicus commented:
@rodica-andronache 

I've fixed masonry on safari and now the thumbnail hides if it's disabled in the content ordering :+1: Thanks for the suggestion! 

Also fixed the content width when using full content inside covers layout posts. Let me know if everything else arises :dancers:

rodica-andronache commented:
@abaicus tested all the options again, plus migrations, and the only remaining issues I've seen are:

- [ ] Disabling `Thumbnail` from `Post Content Order` in `List` mode doesn’t hide the featured image
- [ ] for https://github.com/Codeinwp/neve-pro-addon/pull/897 this
`for Covers layout, the text from the Read More button is the one from the Text Color option, not from Buttons, as it should, I think.` -> The button color still not applying, it remains white
![1](https://user-images.githubusercontent.com/3266155/94415517-4cc29c80-0186-11eb-9c5b-c3248545ecc8.png)

abaicus commented:
@rodica-andronache I've fixed the issue with the button controls that were not applying. It should be good.
